ompi/mca/osc/base/osc_base_frame.c

```diff
@@ -36,6 +36,24 @@
 
 #include "ompi/mca/osc/base/static-components.h"
 
+void
+ompi_osc_base_set_memory_alignment(struct opal_info_t *info,
+                                   size_t *memory_alignment)
+{
+    int flag;
+    opal_cstring_t *align_info_str;
+
+    opal_info_get(info, "mpi_minimum_memory_alignment", &align_info_str, &flag);
+    if (flag) {
+        long long tmp_align = atoll(align_info_str->string);
+        OBJ_RELEASE(align_info_str);
+        if (0 < tmp_align && ((long long) OPAL_ALIGN_MIN < tmp_align)) {
```

Comment:
@bwbarrett I don't think there's a case where OPAL_ALIGN_MIN < 0 - does this look right?

Reply:
I agree on the OPAL_ALIGN_MIN and I think you can make the assumption that OPAL_ALIGN_MIN is relatively small, which means you could simplify this to `if ((long long) OPAL_ALIGN_MIN < tmp_align))`.

Reply:
Thanks - fixed and squashed.